virtual_workstation_setup.sh

```diff
@@ -143,9 +143,6 @@ setup_master() {
 
     git clone git@github.com:10gen/mongo.git
     pushd "$workdir/mongo"
-        mkdir -p src/mongo/db/modules
-        git clone git@github.com:10gen/mongo-enterprise-modules.git src/mongo/db/modules/enterprise
```

Comment:
we haven't done this for master yet, if we remove it from the script we should add the manual steps to the doc?

Reply:
We are merging it tonight. I am just getting this ready